packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### General
+
+-   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect), or [checkbox](#/components/Checkbox) instead.
```

Comment:
I think we should refer to Numeric Input instead. They are closer in terms of intent then checkbox and toggles. 
(and sorry if this is the third time you see this message, seems like it doesn't want to show up)

Reply:
Ok. And how would you choose between Numeric Input and Slider?